src/app/communities/create/page.tsx

```diff
@@ -52,6 +52,20 @@ export default function CreateCommunityPage() {
       } else if (errorMessage === 'User not authenticated') {
         // Open the authentication popup
         setAuthPopupOpen(true)
+      } else if (data?.toxic) {
```

Comment:
ideally we should handle this as an error message itself (rather than checking explicitly for `data?.toxic`) but we can simplify this in a later PR